tests/unit/test_operations/test_vectorize.py

```diff
@@ -99,32 +99,3 @@ def encode(self, value):
             "Stage_customtextencoder_vector_",
         ]
         assert all(vector in dataset.schema for vector in vectors)
-
-    def test_vector_search(self, test_dataset: Dataset):
```

Comment:
we have these tests elsewhere right?

Reply:
No - these tests are no longer useful given we are now losing the `services` endpoints.